src/plugins/gui/providers/bityProvider.ts

```diff
@@ -490,6 +499,28 @@ const completeSellOrder = async (approveQuoteRes: BityApproveQuoteResponse, core
   }
 
   const spendInfo: EdgeSpendInfo = {
+    savedAction: {
+      type: 'sell',
```

Comment:
Going along with my core review, I see that you are saving a `sell` action, but not a `sellNetworkFee` action. I really don't think the `sellNetworkFee` action does anything, or makes sense to even save.

Reply:
The `sellNetworkFee` is used for token transactions

Reply:
Maybe `sellNetworkFee` and `Token Sell Network Fee` is more appropriate then. I thought it was kind of strange with the "Sell Network Fee" string.